api/src/integration/java/org/pmiops/workbench/FireCloudIntegrationTest.java

```diff
@@ -133,7 +133,7 @@ public void testStatusDev() throws Exception {
   }
 
   /**
-   * Ensure swe can successfully use delegation of authority to make FireCloud API calls on behalf
```

Comment:
Apologies in advance for the unrelated typo fix. I find myself seeing these things, and not wanting to lose state on current work (e.g. stashing changes, creating a new branch, creating a separate PR... altogether takes me 5-10 mins) and end up making the documentation fix within the same branch. I'll try to catch myself earlier next time... if folks have suggestions on how they handle this, I'm all ears.